src/P5Functions/Math.js

```diff
@@ -433,7 +433,14 @@ module.exports = function withMath(p, undef) {
       return internalRandomGenerator() * arguments[0];
     }
     var aMin = arguments[0], aMax = arguments[1];
-    return internalRandomGenerator() * (aMax - aMin) + aMin;
+    while (true) {
+      var ir = internalRandomGenerator();
+      var result = ir * (aMax - aMin) + aMin;
+      if (result !== aMax || result === aMin) {
```

Comment:
the second part of this conditional will only ever be true if the first part is, too, unless min and max are the exact same value, which would be  a programmer error (`random(0.3, 0.3)` for instance, which isn't a random value but the fixed value 0.3), so you might want this rephrased as
```
if (result >= aMin && result < aMax) {
  return result;
}

```